content/partials/examples/dropdown.slim

```diff
@@ -798,7 +798,7 @@
       .ui.green.empty.circular.label
       |  Tag D
 .ui.hidden.divider
-|  Show me posts trending
+span Show me posts trending 
```

Comment:
It's an interesting question. Let me explain it here: in the original version, this string "Show me posts trending" is naked in the ".example" div, so if you click the code icon to generate code and example segment, the string without any class or tag will be filtered out from the example segment, it will appear right down the header "Dropdown" without any decoration. So is the problem with the string "User Avatar" in image example. As to the white space, it is added for the sentence "Show me posts trending today", "today" is another single bold div with a dropdown, if no whitespace added here, "trending" will be right next to "today".